### PR TITLE
feat(fdi-gateway): test using aad-pod-identity

### PR DESCRIPTION
**Pull request by jumana-s**
Test retrieving storage account credentials using aad-pod-identity

### CODE REVIEW

File: storage-system/fdi-gateway-system/minio.jsonnet [hunk 1]
```diff
@@ -63,6 +63,28 @@ local values = |||
       volumeMounts:
       - mountPath: /custom
         name: minio-sh
+    sidecars:
```
Comment (blairdrummond) [inline storage-system/fdi-gateway-system/minio.jsonnet:66]: I don't think `sidecars:` exists. Sidecars just go under the `containers:` array

File: storage-system/fdi-gateway-system/minio.jsonnet [hunk 2]
```diff
@@ -52,6 +50,8 @@ local values = |||
       - -c
       - |
         echo '#!/bin/sh' > /custom/minio.sh
+        echo 'sleep 90'
+        echo 'source /custom/creds.env'
```
Comment (blairdrummond) [inline storage-system/fdi-gateway-system/minio.jsonnet:54]: Better to have a loop that checks if the creds.env file has been populated (look at the while-curl block)

**Review by blairdrummond — CHANGES_REQUESTED** [review]

File: storage-system/fdi-gateway-system/minio.jsonnet (hunk 1 above)
Comment (jumana-s) [reply storage-system/fdi-gateway-system/minio.jsonnet:66]: The Minio Helm chart doesn't seem to have `containers:`, `sidecars` is there though.  https://github.com/bitnami/charts/tree/master/bitnami/minio

Comment (blairdrummond) [reply storage-system/fdi-gateway-system/minio.jsonnet:66]: Right! This is in the values not the deployment, good call

**Comment by jumana-s** [comment]
I will create a new PR, since fdi-gateway-system has been refactored to use kustomize.

**Pull request closed without merging**